src/tests/test_pkcs11_high_level.cpp

```diff
@@ -833,11 +837,11 @@ Test::Result test_rsa_sign_verify()
    std::vector<uint8_t> plaintext(256);
    std::iota(std::begin(plaintext), std::end(plaintext), 0);
 
-   auto sign_and_verify = [&keypair, &plaintext, &result](const std::string& emsa, bool multipart) -> void
+   auto sign_and_verify = [&keypair, &plaintext, &result](const std::string & emsa, bool multipart) -> void
```

Comment:
astyle because of unknown reason think that this is not function parameter but binary and operation ;) 

hack for this:
in general use : --pad-oper (to fix missing spaces around operators), then manually fix this kind of issues, with disabled --pad-oper (which should be disabled in astylerc)